packages/core/docs/general/where-to-start.md

```diff
@@ -0,0 +1,45 @@
+# Where to start?
+
+This document will show where you should start if you are:
+
+[[toc]]
+
+[comment]: <> (## Developer creating a shop)
+
+## Integrator
+
+All integrations created and maintained by Core Team and our partners you can find on the special created [integrations](../integrations/) page.
+
+Didn't find the integration you were looking for? You can create your own eCommerce, CMS, or other integrations with Vue Storefront but remember about:
+
+::: warning Want to build an integration?
+If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io) server** first. We are eager to help you with building it and ensuring its high quality! Building the integration with the help of the core team is the best way to keep its quality high and making it officially recommended once completed.
+:::
```

Comment:
This warning is already on the pages for creating integrations, so there is no need to have it here as well.

```suggestion
If you didn't find the integration you were looking for, you can create your own.
```